plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/OTelTelemetrySettings.java

```diff
@@ -60,6 +63,34 @@ private OTelTelemetrySettings() {}
         Setting.Property.Final
     );
 
+    /**
+     * Probability of sampler
+     */
+    public static final Setting<Double> OTEL_TRACER_HEAD_SAMPLER_SAMPLING_RATIO = Setting.doubleSetting(
```

Comment:
We could move it to `TelemetrySettings` instead and make it independent of OTEL
 
```suggestion
    public static final Setting<Double> TRACER_SAMPLER_PROBABILITY = Setting.doubleSetting(
    "telemetry.otel.tracer.head.sample.sampling.ratio",
```

Reply:
Updated in new commit.